### ISSUE TITLE
[Implement]

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-dynamic-planning#71]
This is a comment related to the page [Implement](//w3c.github.io/wai-dynamic-planning/implement/index.html)

The activities section is good but relies too much on "Ensure."  Suggest rewording first paragraph as follows:

It is important that accessibility related tasks are integrated across various roles, that they are fully understood, and that the team has the capability to meet expectations. Deliverables and assignments should be specific to the role. For example, development of a procurement policy that includes accessibility may be assigned to administrators, while project managers may be assigned the task of creating a clear statement of relevant accessibility coding guidelines and how to meet them. 

It adds 5 words but is much clearer IMO and avoids duplicating the dread "Ensure" word. Also may allow the removal entirely of this very muddy paragraph:

For those involved in delivering to specific standards, ensure that they have a clear definition of the applicable elements. For example, designers and developers need to know the specific success criteria they need to address and how to implement the appropriate techniques to complete the work. Ensure that everyone has the resources to help them with their tasks.

**Comment by iadawn** [comment]
I have taken your suggestions and rewritten the section with them in mind. This is available in 2d110c8.

I tried to present three core messages:
- Make sure folks know what they are doing and what to do if they have a problem
- Be specific about what they are doing and be aware that responsibility may be dispersed
- Verify your progress

I cleared out some of the muddy stuff. Hopefully this is moving in the right direction?
